app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
+    class GreetingsController < ApplicationController
+      def show
+        @greeting = Greeting.find(rand(1..5))
+        render json: @greeting, status: :ok
+      end
```

Comment:
- [ ] Here is the project requirement 👇🏻Kindly Make you create 5 different greetings, for the table for storing your messages, Actually you can use the seed to create five different greetings.

![image](https://user-images.githubusercontent.com/79058364/216478580-31fba987-8dbc-4d98-a4d2-c3194ab2609e.png)

Reply:
- [ ] @BenMukebo , I get the same result when I run the app using `rails s` but when I used `./bin/dev` I get the result.